command/core.py

```diff
@@ -46,3 +50,18 @@ def handle_team_join(self, event_data):
             logging.info(new_id + " added to database - user notified")
         except SlackAPIError:
             logging.error(new_id + " added to database - user not notified")
+
+    def get_help(self):
+        """Get help messages and return a formatted string for messaging."""
+        message = {"text": "Displaying all available commands. "
+                           "To read about a specific command, use "
+                           "\n`/rocket [command] help`\n",
+                   "mrkdwn": "true"}
+        attachments = []
+        for cmd in self.__commands.values():
+            cmd_name = cmd.get_name()
+            cmd_text = "*" + cmd_name + ":* " + cmd.get_desc()
+            attachment = {"text": cmd_text, "mrkdwn_in": ["text"]}
+            attachments.append(attachment)
+        message["attachments"] = attachments
+        return json.dumps(message)
```

Comment:
We should be using `flask.jsonify` instead of `json.dumps` so that we can get the correct headers for the response.

Reply:
done !